app/Services/PostService.php

```diff
@@ -88,6 +104,13 @@ public function destroy($postId)
             'id' => $postId,
         ]);
 
-        return $this->repository->destroy($postId);
+        $trashed = $this->repository->destroy($postId);
+
+        // Dispatch job to send post to Quasar
+        if (config('features.pushToQuasar')) {
+            SendPostToQuasar::dispatch(Post::withTrashed()->find($postId));
```

Comment:
This should be `SendDeletedPostToBlink`, right?

Reply:
@DFurnes it sure should be!